ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,28 @@
+passenger=(12, True, "Bonnell, Miss. Elizabeth", "female", 58)
+print (passenger)
+name= print ("name:",(passenger[2]))
+age= print("age:", (passenger[-1]))
```

Comment:
Make sure you're assigning the value to the variable _first_, then `print`ing that variable _second_:

---

![image](https://user-images.githubusercontent.com/16143771/165650805-aac9722c-7d3c-4822-8cdc-47ea91cc6654.png)